cmd/archiveconverter/main.go

```diff
@@ -94,7 +94,10 @@ func processFile(file File, outputFile string) {
 	} else {
 
 		if err := proto.Unmarshal(content, snapshot); err != nil {
-			log.Fatalf("Failed to parse protobuf message: %v", err)
+			snapshot = &pb.SnapshotWithType{}
```

Comment:
This change makes possible to process both protos with ChangeSnapshot root and SnapshotWithType